### PR TITLE
remove SYSTEM_KERNEL_DIRS

### PR DESCRIPTION
**Pull request by minrk**
use SYSTEM_JUPYTER_PATH from core instead

sibling to jupyter/jupyter_notebook#64

### CODE REVIEW

File: jupyter_client/kernelspec.py [hunk 1]
```diff
@@ -136,8 +122,8 @@ def _get_destination_dir(self, kernel_name, user=False):
         if user:
             return os.path.join(self.user_kernel_dir, kernel_name)
         else:
-            if SYSTEM_KERNEL_DIRS:
-                return os.path.join(SYSTEM_KERNEL_DIRS[-1], kernel_name)
+            if SYSTEM_JUPYTER_PATH:
+                return os.path.join(SYSTEM_JUPYTER_PATH[0], 'kernels', kernel_name)
             else:
                 raise EnvironmentError("No system kernel directory is available")
```
Comment (takluyver) [inline jupyter_client/kernelspec.py]: This case shouldn't be necessary now, right? We're ensuring that there's always at least 1 path in `SYSTEM_JUPYTER_PATH`.

Comment (minrk) [inline jupyter_client/kernelspec.py]: Right.

**Comment by Carreau** [comment]
same, check for `SYSTEM_KERNEL_DIRS` and warn if set ?

**Comment by minrk** [comment]
same as before - how would it be set? It's a constant defined in this module.

**Pull request merged**